uberfire-project/uberfire-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/ModuleRepositoryResolverImpl.java

```diff
@@ -171,6 +182,37 @@ public ModuleRepositoryResolverImpl(final @Named("ioStrategy") IOService ioServi
         return repositories;
     }
 
+    private String getGlobalRepoPath() {
+        ArtifactRepositoryPreference artifactRepositoryPreference = new ArtifactRepositoryPreference();
+        artifactRepositoryPreference.defaultValue(artifactRepositoryPreference);
+        String global = artifactRepositoryPreference.getGlobalM2RepoDir();
+        if (global == null) {
+            global = "repositories" + File.separator + "kie" + File.separator + "global";
```

Comment:
can't we just reuse an existing default? even better, couldn't we just move this default to the `artifactRepositoryPreference.getGlobalM2RepoDir`? and be able to reuse it also by M2ServletContextListener?

Reply:
Ok